### ISSUE TITLE
[Adaptive Cards-Food Order.json]: Keyboard tab focus is not shifted to newly opened dialog after activating “Steak" button on 'FoodOrder.json' card.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9243]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0

**Pre-requisites:**
Navigate using keyboard only.

**Repro steps:**

1. Open the Adaptive Card Visualizer application.
2. Navigate to FoodOrder.Json button by using Tab key and activate it.
3. FoodOrder adaptive card will be appeared on right sight of the pane.
4. Navigate to FoodOrder card by using Tab key.
5. Navigate to "Steak" button and activate it.
6. Observe the keyboard issue.

**Actual Result:**
Keyboard tab focus is not shifted to newly opened dialog after pressing “Steak" button.

**Note**
Same issue is also repro with "Chicken" and "Tofu" buttons on 'FoodOrder.json' card.

**Expected Result:**
Keyboard tab focus should be shifted to the next interactive element i.e., "Rare" radio button present under newly opened dialog after pressing “Steak" button.

**User Impact:**
Keyboard Users might get affected as keyboard focus is moving to the next button "Chicken" when “Steak" button is selected. Due to this behavior, user are getting impacted as they have to press more tabs to reach back the newly opened dialog.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/661481ac-27c8-41f1-8c58-74bdaef4868c

### Card JSON

```JSON
N/A
```
